common/traits/druid_forms.txt

```diff
@@ -60,8 +60,8 @@ pack_form = {
 	martial = 4
 	diplomacy = -5
 	command_modifier = {
-		damage = 0.3
-		morale_offence = 0.3
+		damage = 0.25
+		morale_offence = 0.25
```

Comment:
I would even add a decrease in Kaldorei opinion (or general, I dunno) due to Malfurion's prohibiton of the pack form

Reply:
it would be more fitting for pack_form_mastered trait and i will probably add it when i get to making the pack form avaible